code/datums/martial/cqc.dm

```diff
@@ -8,6 +8,14 @@
 	name = "CQC"
 	help_verb = /mob/living/carbon/human/proc/CQC_help
 	block_chance = 75
+	var/just_a_cook = FALSE
+	var/static/list/areas_under_siege = typecacheof(list(/area/crew_quarters/kitchen,
+														/area/crew_quarters/cafeteria,
+														/area/crew_quarters/bar))
+
+/datum/martial_art/cqc/under_siege
+	name = "Culinary School Training"
```

Comment:
>Culinary School Training

Why isn't this Close Quarters Cooking

Reply:
Holy shit

Reply:
Snake, try-a to remember the basics of-a CQC! I don't-a care what-a the crab tastes like, you have-a Greeks to fight!